lisp_ease/CMakeLists.txt

```diff
@@ -0,0 +1,199 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
Please get the cram7 state of code, test if everything still works. Try to change pull request from cram2:master to cram2:cram7. If impossible, work on all other comments first, let Gaya take a look, then close PR and reopen onto cram7.